test/using-bare-variables-success.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - hosts: localhost
-  become: no
```

Comment:
All changes in this file are not related to the changes implemented in the PR and should not be part of the changeset.